src/main/java/mate/academy/Main.java

```diff
@@ -27,7 +35,8 @@ public static void main(String[] args) {
         secondCinemaHall.setCapacity(200);
         secondCinemaHall.setDescription("second hall with capacity 200");
 
-        CinemaHallService cinemaHallService = null;
+        CinemaHallService cinemaHallService =
+                (CinemaHallService) injector.getInstance(CinemaHallService.class);;
```

Comment:
There is an extra semicolon at the end of the line. Please remove it to avoid potential issues and improve code readability.